static/css/foundation.css

```diff
@@ -3733,11 +3733,27 @@ a.thumbnail {
     margin-left: 0; }
 
 .column, .columns {
-  -ms-flex: 1 1 0px;
-      flex: 1 1 0px;
-  padding-right: 0.9375rem;
-  padding-left: 0.9375rem;
-  min-width: 0; }
+  width: 100%;
```

Comment:
This'll be overridden again after next upgrade of the Foundation Library. I'd rather like to have it added to our app.css file instead, with a remark that it's necessary (right now) to keep the site working.

Reply:
You won't need to modify the minified version either that way 👍